assets/js/romo/base.js

```diff
@@ -699,9 +699,11 @@ Romo.prototype.array = function(value) {
     typeof(object)          === 'function' &&
     typeof(object.nodeType) !== 'number'
   );
+  var isSelect = (object.nodeName.toLowerCase() === 'select')
```

Comment:
@kellyredding - You might have to do a check for `object.nodeName` to make sure its not `undefined` before calling `toLowerCase`.

Reply:
@jcredding good catch.